### PR TITLE
Multilang

### PR DESCRIPTION
**Pull request by contributorpw**
This is related to #87 

- [x] Add the translations menu list as a part of the main menu. On partials
- [x] Add the langs menu list as a part of the main menu. On partials
- [x] Basic i18n support
- [ ] Add drop-down list wrapping
- [ ] Translate the example site to Russian 

:children_crossing: Requires customization of the menu. Lists are not wrapped in a drop-down list. I need help for this.

<details><summary><b>Video example</b></summary>

screenrecord

</details>

<details><summary><b>How to debug</b></summary>
<br >

1. Pull the PR branch
1. Checkout
1. `cd ./`
1. Run `hugo`

```sh
$> cd ./exampleSite && rm -rf ./public && hugo serve --config ./config.toml --themesDir ./../.. --renderToDisk --disableFastRender
```

</details>

### CODE REVIEW

File: exampleSite/config.toml [hunk 1]
```diff
@@ -14,6 +14,17 @@ enableGitInfo = true
 # pygmentsStyle = 'monokailight'
 pygmentsCodeFences = true
 
+# Multilang config
+DefaultContentLanguage = 'en'
```
Comment (alex-shpak) [inline exampleSite/config.toml]: Please rename to camel case in relevant places
e.g. `defaultContentLanguage`, I know that in docs it's upper camel case, but historically I used camel case there.
So just to keep it consistent.

File: exampleSite/config.yaml [hunk 2]
```diff
@@ -14,6 +14,17 @@ enableGitInfo: true
 # pygmentsStyle: monokailight
 pygmentsCodeFences: true
 
+# Multilang config
+DefaultContentLanguage: en
```
Comment (alex-shpak) [inline exampleSite/config.yaml]: `defaultContentLanguage`

**Comment by alex-shpak** [comment]
Thank you for work!
I have idea to place language switch either in the footer of page or next to title as globe icon.

I would ask you to do few changes prior to merge:
- Please remove `resources` changes from PR. Especially `resource` folder in root.
- Rename `DefaultContentLanguage` to `defaultContentLanguage` in config files

File: exampleSite/config.toml (hunk 1 above)
Comment (contributorpw) [reply exampleSite/config.toml]: Done

**Comment by contributorpw** [comment]
- [ ] remove resources changes from PR. Especially resource folder in root.
- [ ] Rename DefaultContentLanguage to defaultContentLanguage in config files

> I have idea to place language switch either in the footer of page or next to title as globe icon.
This great! But I still can’t figure out how to make a pretty beautiful drop-down list like (for example) the button shortcode.

**Comment by alex-shpak** [comment]
Merging to `feature/multilang`

**Pull request merged**